start

```diff
@@ -114,6 +119,9 @@ function process_args() {
     --enable-soroban-diagnostic-events)
       ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=true
       ;;
+    --enable-soroban-rpc-admin-endpoint)
+      ENABLE_SOROBAN_RPC_ADMIN_ENDPOINT=true
```

Comment:
Actually, why do we need the option? Horizon exposes the same type of admin port all the time, without an option to enable / disable it?

If there's no strong requirement to have it be an option, I'd suggest starting with it always being enabled just like Horizon. Less options is less branches to maintain and test.

Reply:
That would make things simple for sure!
The only consideration that I can think of is about functionality that the port provides. If it's just read-only access to metrics it should be fine. But if it allows some kind of RW access maybe it should be possible to disable it just in case.

Reply:
If there's an advantage to making them optional sgtm to keep it.

Reply:
I considered that at first, as like the convention over config also, rpc service disables the admin port by default, I was a bit concerned about ignoring that, at present it is just read only metrics and pprof output. what if we default ENABLE_SOROBAN_RPC_ADMIN_ENDPOINT=true here in quickstart? provides same zero-additional config outcome for quickstart users, but people can override if concerned.

Reply:
Most people won't need it. I think what you've got is fine. Maybe we even want to add it for Horizon.